core/src/plugins/filed/python/postgresql/bareos-fd-postgresql.py

```diff
@@ -1668,11 +1669,11 @@ def end_backup_job(self):
         Make sure that db connection was closed in any way,
         especially when job was canceled
         """
-        if self.full_backup_running:
+        bareosfd.DebugMessage(100, "end_backup_job() entry point in Python called\n")
+        # Execute pg_backup_stop() in case of incremental
+        if self.backup_running:
```

Comment:
Would `backup_running` not be better called `incremental_backup_running` if it determines if the backup is an incremental or not?

Reply:
Or maybe an single variable of  enumeration type would make more sense which contains
- not_running
- incremental_running
- full_running
This could be easily extended to contain diff also in the future.

Reply:
This need improvement as it can create confusion.
we will introduce a variable backup_full which indicate if the backup level is Full or not and a pg_backup_running False/True to indicate if a pg_backup_start has been issued.